docs/manual/src/swift/configuration.md

```diff
@@ -9,11 +9,11 @@ more likely to change than other configurations.
 
 | Configuration name                  | Default                  | Description                                                                                                                                                        |
 | ----------------------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `cdylib_name`                       | `uniffi_{namespace}`[^1] | The name of the compiled Rust library containing the FFI implementation (not needed when using `--library`).                                              |
+| `cdylib_name`                       | `uniffi_{namespace}`[^1] | The name of the compiled Rust library containing the FFI implementation (not needed when using `generate --library`).                                              |
 | `module_name`                       | `{namespace}`[^1]        | The name of the Swift module containing the high-level foreign-language bindings.                                                                                  |
 | `ffi_module_name`                   | `{module_name}FFI`       | The name of the lower-level C module containing the FFI declarations.                                                                                              |
 | `ffi_module_filename`               | `{ffi_module_name}`      | The filename stem for the lower-level C module containing the FFI declarations.                                                                                    |
-| `generate_module_map`               | `true`                   | Whether to generate a `.modulemap` file for the lower-level C module with FFI declarations.  Not valid when using `--library`, in library-mode the modulemap is always generated.
+| `generate_module_map`               | `true`                   | Whether to generate a `.modulemap` file for the lower-level C module with FFI declarations. (ignored by `uniffi-bindgen-swift`)                            | 
```

Comment:
Can we add an example command for both a Swift and a `.Xcframework`?

Reply:
I added a bunch to `docs/manual/src/swift/uniffi-bindgen-swift.md`.